micrometer-registry-newrelic-telemetry/deployment/src/test/java/io/quarkiverse/micrometer/registry/newrelic/telemetry/deployment/NewRelicEnabledTest.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkiverse.micrometer.registry.newrelic.telemetry.deployment;
+
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.newrelic.telemetry.micrometer.NewRelicRegistryConfig;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class NewRelicEnabledTest {
+    static final String REGISTRY_CLASS_NAME = "com.newrelic.telemetry.micrometer.NewRelicRegistry";
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .withConfigurationResource("test-logging.properties")
+            .overrideConfigKey("quarkus.micrometer.binder-enabled-default", "false") // FIXME not sure about this
+            .overrideConfigKey("quarkus.micrometer.export.newrelic.enabled", "true")
+            .overrideConfigKey("quarkus.micrometer.export.newrelic.publish", "false")
+            .overrideConfigKey("quarkus.micrometer.export.newrelic.apiKey", "dummy")
+            .overrideConfigKey("quarkus.micrometer.export.newrelic.uri", "http://localhost")
+            .overrideConfigKey("quarkus.micrometer.export.newrelic.accountId", "dummy")
```

Comment:
There is no ".telemetry" segment in these configurations.. which means we have a mismatch in config. The config object is just looking for the newrelic prefix, so it finds these values.. but these values don't match what the docs say you're looking for (newrelic.telemetry)

Reply:
This is what was causing the collisions in the documentation... We cannot have 2 projects with the same config properties, with this setup.
I would prefer not to break what we have now... 
What if we place the previous properties in the telemetry project and add another prefix to the deprecated project?

Reply:
I chose to keep the new newrelic.telemetry.* structure.
Adding the native mode tests...